src/models/supplemental_accessors.jl

```diff
@@ -14,12 +14,12 @@ _remove_aggregration_topology!(bus::Bus, ::Area) = bus.area = nothing
 """
 Calculate the admittance of AC branches
 """
-get_series_susceptance(b::ACBranch) = 1 / get_x(b)
-get_series_susceptance(b::TapTransformer) = 1 / (get_x(b) * get_tap(b))
+get_series_susceptance(b::ACBranch) = -1 / get_x(b)
+get_series_susceptance(b::TapTransformer) = -1 / (get_x(b) * get_tap(b))
```

Comment:
This function should be 
```suggestion
function get_series_susceptance(b::TapTransformer)
    y = 1 / (get_r(b) + get_x(b) * 1im)
    y_a = y / (get_tap(b))
    return imag(y_a)
end
```